roles/openshift_storage_nfs/handlers/main.yml

```diff
@@ -4,3 +4,8 @@
     name: nfs-server
     state: restarted
   when: not (nfs_service_status_changed | default(false))
+
+- name: restart nfs-config
+  service:
+    name: nfs-config
```

Comment:
Looks like the handler may not be enough. If there was a failed nfs-server instance, then nfs-config needs to be run before service start of nfs-server in tasks/main.yml